docs/Contribute_To_Frontend_Quick_Guide.md

```diff
@@ -0,0 +1,243 @@
+
+## In a browser, go to the IML location
+* [https://ct7-adm.lfs.local:8443](https://ct7-adm.lfs.local:8443)
+
+## Verify that the small change worked.
+
+It is possible that the browser cache may require refreshing for the change to take place.
+
+* Click on 
+    * Configuration 
+    * Verify that the pull down menu has the item: "My Servers"
+    * The image below also shows a change for "My Power Control"
+
+![iml_flow](md_Graphics/config_my_servers.png)
+
+---
+# Congratulations! You just made a change to the IML Frontend code.
+---
+
+## The process outlined above is the basic technique for modifying the Frontend IML code.
+
+## A note about starting and stopping chroma-config
+* The amount of time to complete 
+    * chroma-config start
+    * chroma-config stop
+
+* can be time consuming, however, the commands are still an effective way to ensure that all processes start and stop.
+
+* A faster method to restart just the "view_server" process can be accomplished as follows:
+
+    * **supervisorctl -c /usr/share/chroma-manager/production_supervisord.conf restart view_server**
+
+
+* To check the IML services status
+
+   * **supervisorctl -c /usr/share/chroma-manager/production_supervisord.conf status**
+
+
+# Pushing the code change to github
+
+### On your local machine, i.e., not the vagrant VM:
+
+* cd ~/vagrant-projects/vhpc/GUI
+
+Ensure you are on the proper branch
+* git branch
+
+Otherwise, change to the **my-fix** branch
+* git checkout my-fix
+
+Add and Commit the change.
+
+* git status
+
+* git add source/iml/app/app-states.js
+
+* git commit -s 
+
+This will cause the **eslint**, **flow** and **unit** tests to run.
+
+Add the following comment:
+
+```
+This is a test fix for the frontend
+
+ - Changed the Configuration dropdown label for Servers
+```
+
+* Save the commit 
+
+## Sync with the github origin
+Prior to pushing the code change it may be necessary to synchronize your cloned repo with the master repo that may have changed while you were working.
+
+To synchronize your cloned master with the origin master and to rebase your changed on top of the changes of everyone else, do the following:
+
+|Git task|Git command|
+|--------|--------------|
+|  List the Current origin  |  git remote -v |
+|  Switch to the master branch |git checkout master |
+| Fetch and Rebase | git pull  --rebase |
+| Switch back to your branch |  git checkout my-fix |
+| Put your changes on top of everyone elses | git rebase master |
+
+## Push to github
+* git push origin my-fix
+
+## Create a Pull Request
+* In a Browser, go here: [https://github.com/intel-hpdd/GUI](https://github.com/intel-hpdd/GUI)
+* Click on the **Branches** tab.
+* Find the branch named **my-fix**
+* On the right hand side, click the button for **New pull request**
+
+* Fill out a comment with the description of the change
+* Choose Reviewers, Labels, Assignees, etc.
+* Click **Create pull request**
+
+## Copy the URL and request a code review via email
```

Comment:
Probably better to either assign reviewers directly or request in a comment on the PR.

Reply:
Fixed